### DISCUSSION TITLE
IPerf3 shows inaccurate numbers on Transfer Bytes (comparing with wireshark)

### DISCUSSION THREAD

**Discussion by mzazure** [esnet/iperf#1635] [General]
see below screenshot.  Can someone help explain this?
![image](https://github.com/esnet/iperf/assets/144375901/3d13b6f4-a4c6-4644-b747-d4988fe87965)

**Comment by davidBar-On** [comment]
By default, Wireshark calculates the SEQ per the packets it captures.  Therefore, when not all packets are captured, SEQ will not be the actual number.   See this [StackExchange explanation](https://superuser.com/questions/828294/how-can-i-get-the-actual-tcp-sequence-number-in-wireshark), including how to view the real SEQ number.

In this case, as there are 240ms between packets 14 and 19, probably Wireshark did not capture many of the packets during this period of time.

**Reply by mzazure** [reply]
Thanks for the clarification! I realized that the numbers are aligned when the payload is big
